### ISSUE TITLE
Unable to update from 1.11 to 1.14

### ISSUE THREAD

**Issue by persapiens** [franzbecker/gradle-lombok#48]
I'm using `gradle-lombok` and `querydsl` plugins. If I update `gradle-lombok` from 1.11 to new version, build fails at `compileQuerydsl` task.

Sample project is [here.](https://gitlab.devops.ifrn.edu.br/corporativo/conta)

**Comment by persapiens** [comment]
Any idea?

**Comment by franzbecker** [comment]
Hey there,

this does not seem to be a problem with the plugin but with Lombok <-> QueryDsl. I removed the plugin from your project and added Lombok manually, same issue.

What happened between 1.11 and 1.12 is that the lombok dependency was moved from the `compile` scope to `compileOnly` (as it should be, and later also added as an annotation processor).

The diff between both releases: https://github.com/franzbecker/gradle-lombok/compare/v1.11...v1.12

However I could make your project work adding the following lines (in the `build.gradle`):

```
configurations {
    querydsl.extendsFrom compileClasspath
}
```

See https://github.com/ewerk/gradle-plugins/issues/39#issuecomment-197308241

Does that work for you?
